buildSrc/src/main/kotlin/plugin/AndroidLibraryPlugin.kt

```diff
@@ -25,7 +25,8 @@ class AndroidLibraryPlugin : BasePlugin() {
         get() = {
             implementation(
                 Library.daggerHiltAndroid,
-                Library.coroutines
+                Library.coroutines,
+                Library.material,
```

Comment:
Don't add material design here. They are non ui modules that use this plugin

Reply:
Ohh
Should I create a new (second) android library plugin. There's a library module that uses material design

Reply:
You can add it directly to the library's gradle file.
Always consider this when adding dependencies

Reply:
Thanks alot.. It totally skipped my mind that I can just do that